### PR TITLE
fixedEslint_Sec_Time

### PR DESCRIPTION
**Pull request by Bodyan322**

### CODE REVIEW

File: src/tests/constructor.test.js [hunk 1]
```diff
@@ -7,19 +7,19 @@ describe('tests for constructor', () => {
...
-    expect(arr1.length).toEqual(0);
-    expect(arr2.length).toEqual(5);
-    expect(arr3.length).toEqual(2);
-    expect(arr3[0]).toEqual(1);
-    expect(arr3[1]).toEqual(2);
+    expect(arr1.length).toBe(0);
+    expect(arr2.length).toBe(5);
+    expect(arr3.length).toBe(2);
+    expect(arr3[0]).toBe(1);
+    expect(arr3[1]).toBe(2);
```
Comment (PodstDO) [inline src/tests/constructor.test.js:14]: Use `toBe` is a good practice in this case.

File: src/tests/constructor.test.js [hunk 2]
```diff
@@ -7,19 +7,19 @@ describe('tests for constructor', () => {
...
     expect(arr4.length).toBe(1);
-    expect(arr4[0]).toEqual('1');
+    expect(arr4[0]).toBe('1');
   });
 
   test('should have own property length', () => {
     const arr = new MyArray();
 
-    expect(arr.hasOwnPropObject.prototype.hasOwnProperty.callerty('length')).toBeTruthy();
+    expect(Object.prototype.hasOwnProperty.call(arr, 'length')).toBeTruthy();
```
Comment (PodstDO) [inline src/tests/constructor.test.js:22]: Amazing!😍💥

File: src/tests/forEach.test.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import MyArray from '../index.js';
 
 
 describe('tests for method forEach', () => {
-  let arr = 0;
+  let arr = null;
```
Comment (PodstDO) [inline src/tests/forEach.test.js:5]: Great! 👌

**Review by PodstDO — APPROVED** [review]
**You're the best!**

**Review by Vladyslav-Shvorobuk — APPROVED** [review]

**Pull request merged**